website/docs/tutorials/guides/dependencies-management.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_label: Dependencies (2 min)
+---
+
+# Dependencies
+
+Protostar manages dependencies (packages) using [git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) ([similarly to Foundry](https://onbjerg.github.io/foundry-book/projects/dependencies.html)). This is the reason Protostar expects [git](https://git-scm.com/) to be installed. Keep this in mind, especially if your project uses git submodules.
+
+:::note
+Using Git submodules as a foundation for package management is not an ideal approach. Therefore, [Starkware](https://starkware.co/) plans to create a proper package manager.
+:::
+
+## Adding a dependency
+
+To add a dependency, run `protostar install` from the [working directory](https://en.wikipedia.org/wiki/Working_directory) of your project:
+
+```console
+$ protostar install https://github.com/bellissimogiorno/cairo-integer-types
+12:00:00 [INFO] Installing cairo_integer_types (https://github.com/bellissimogiorno/cairo-integer-types)
+```
+`lib` folder should now contain the installed dependency:
+```console
+$ tree -L 2
+.
+├── lib
+│   └── cairo_integer_types
+├── protostar.toml
+├── src
+│   └── main.cairo
+└── tests
+    └── test_main.cairo
+```
```

Comment:
I am not sure if this example output is necessary, I think just a command is more than enough. The protostar itself should provide sufficient feedback about the success.